dist/lib/lib.js

```diff
@@ -16,7 +16,10 @@ const debug = require("debug")("FHIR");
 
 const {
   patientParams
-} = require("./settings");
+} = require("./settings"); // eslint-disable-next-line no-undef
+
+
+const fetch = global.FHIRCLIENT_PURE ? window.fetch : require("cross-fetch");
```

Comment:
I think this should be
```js
const fetch = global.FHIRCLIENT_PURE ? window.fetch : require("cross-fetch").fetch;
```
Even though `const fetch = require("cross-fetch");` is technically valid, TS won't understand it unless we use `require("cross-fetch").fetch` or `require("cross-fetch").default`.

Also, what is the reason for switching to `cross-fetch`? I see your point for not polluting the global namespace, but in old browsers without fetch support that is exactly what we should do. In other words, this library should act as polyfill in this case and provide `Request`, `Response`, `Headers`, `fetch`, `URL` and `URLSearchParams` to the global scope. Is that the case now?

Reply:
> TS won't understand it unless we use `require("cross-fetch").fetch` or `require("cross-fetch").default`

I don't think this is required, cross-fetch still exposes a CJS export regardless, and `require` semantics don't change regardless of the TypeScript version being used. However, I didn't realize cross-fetch exposes `fetch` as a named export. I'll update it since I prefer to use those anyway.

> what is the reason for switching to cross-fetch? I see your point for not polluting the global namespace, but in old browsers without fetch support that is exactly what we should do.

Since this library uses webpack, there shouldn't ever be much reason to put more than is actually necessary into the global namespace. cross-fetch does leverage the native implementations if they're available, but it won't put them onto `window`. If a project wants to use `fetch` on older browsers outside of the API provided by this library, I think it's more appropriate that they bring their in own polyfill and use the "pure" build instead.

Reply:
I don't think my changes impact that behavior anyway. With Webpack I don't think evaluating `require('whatwg-fetch')` is capable of mutating `window` anyway.